### PR TITLE
core: reviewed tz inheritance on Administrator, Company and User

### PR DESCRIPTION
**Pull request by mmadariaga**
Fixed time zone inheritance on sanitizes

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [ ] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

File: library/Ivoz/Provider/Domain/Model/Administrator/Administrator.php [hunk 1]
```diff
@@ -111,6 +111,15 @@ public function unserialize($serialized)
             ) = unserialize($serialized);
     }
 
+    protected function sanitizeValues()
+    {
+        if (!parent::getTimezone()) {
+            $this->setTimezone(
+                $this->getTimezone()
+            );
```
Comment (Kaian) [inline library/Ivoz/Provider/Domain/Model/Administrator/Administrator.php]: TBH this is hard to understand. It looks like a self assignment.

File: library/Ivoz/Provider/Domain/Model/Administrator/Administrator.php [hunk 2]
```diff
@@ -121,10 +130,14 @@ public function getTimezone()
...
-                ->getDefaultTimezone();
+        $company = $this->getCompany();
+        if ($company) {
+            return $company->getDefaultTimezone();
+        }
+
+        $brand = $this->getBrand();
+        if ($brand) {
+            return $brand->getDefaultTimezone();
         }
```
Comment (Kaian) [inline library/Ivoz/Provider/Domain/Model/Administrator/Administrator.php]: Can we move this to sanitize?

**Comment by ironArt3mis** [comment]
Rebase completed.

**Comment by ironArt3mis** [comment]
Rebase completed.

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**